wct-core/src/main/java/org/webcurator/ui/target/command/ProfileCommand.java

```diff
@@ -676,11 +689,14 @@ public void updateOverrides(ProfileOverrides overrides) {
 
 		overrides.setH3IncludedUrls(stringToList(h3IncludedUrls));
 		overrides.setOverrideH3IncludedUrls(overrideH3IncludedUrls);
+
+		overrides.setOverrideH3RawProfile(overrideRawProfile);
```

Comment:
The Spring framework automatically maps variables (such as overrideRawProfile and rawProfile) from ProfileCommand to/from the request. In order for the Spring mapping to work properly (and seamlessly and transparently), the property needs to have the same name in the jsp and the command class for the given controller (the command class set by setCommandClass in the controller).

Exchanging overrideRawProfile with overrideH3RawProfile and rawProfile with h3RawProfile introduces a point of confusion between the attributes in the request and how they are actually used in the code. In some sense this introduces technical debt because years from now it won't be clear why other parameters have a 1:1 mapping between the request and the controller/command class, but these two do not. It's simply not consistent.

I know there are reasons why this choice was made, but in doing so, it does introduce confusion and does make the code a bit harder to maintain.

Reply:
Yeah, you're right, I'll fix it.